pavelib/tests.py

```diff
@@ -212,6 +212,26 @@ def coverage(options):
     call_task('diff_coverage', options=dict(options))
 
 
+@no_help
+@task
+@needs('pavelib.prereqs.install_prereqs')
+def combine_jenkins_coverage():
```

Comment:
One question I had about this approach:  

Would it make more sense to not keep the `.jenkins-coveragerc` file in the repo, and instead create it in the job definition (so that the job names aren't specified in the repo anywhere)?  We could specify the location of the `.coveragerc` file to use in an env variable or as a command line arg here and keep this more generic.